changelog/README.md

```diff
@@ -0,0 +1,59 @@
+# Multi-file running changelog
+
+To avoid merge conflicts in the CHANGELOG.md file we accepted the multi-file running changelog strategy.
+
+To follow this strategy you should create a `changelog/unreleased/${PR_NUMBER}.json` file for every PR like:
```

Comment:
for that approach, we have to add a changelog in another commit (first create PR, get the PR number, and put the changelog and PR number into the branch).

Can it be modified so that we don't add the PR number manually? Can it be added under the hood when PR has been created?

Reply:
> for that approach, we have to add a changelog in another commit

I like to use the `-f` option but yes, I agree that it can get annoying.

Reply:
I understand that it's not comfortable to write a changelog only after creating PR. I can propose creating an action that will rename the changelog file to ${PR_NUMBER}.md:
1. You create a PR (with changelog file feature_1.md)
2. The action executes and rename the changelog file to ${PR_NUMBER}.md
Action will be executed only on open/reopen/redy for review moments
WDYT

Reply:
lgtm 👍 

how does it works for release branches where we mostly cherry-pick changes (sometimes already released)? ref https://github.com/mapbox/mapbox-navigation-android/pull/6740#issuecomment-1352894603

Reply:
oh it's a good moment, I think we can create a PR to the release branch first. Then, the action will rename the changelog file. Then, we can cherry-pick commits (changes + renaming commit) to the main branch

Reply:
> oh it's a good moment, I think we can create a PR to the release branch first. Then, the action will rename the changelog file. Then, we can cherry-pick commits (changes + renaming commit) to the main branch

Typically we merge into main branch, and then cherry-pick into the release branch(es). Maybe minor detail but I don't think we want it to require release commits first.